src/components/AnchorForAttachmentsOnly/AnchorForAttachmentsOnlyTypes.ts

```diff
@@ -0,0 +1,12 @@
+import {StyleProp, ViewStyle} from 'react-native';
+
+type AnchorForAttachmentsOnlyProps = {
+    /** The URL of the attachment */
+    source: string;
+    /** Filename for attachments. */
+    displayName: string;
+    /** Any additional styles to apply */
+    style?: StyleProp<ViewStyle>;
+};
```

Comment:
```suggestion
type AnchorForAttachmentsOnlyProps = {
    /** The URL of the attachment */
    source?: string;

    /** Filename for attachments. */
    displayName?: string;

    /** Any additional styles to apply */
    style?: StyleProp<ViewStyle>;
};
```

According to the propTypes.